src/components/pages/dashboard/panels/alerts/alertsPanel.js

```diff
@@ -31,10 +35,17 @@ export class AlertsPanel extends Component {
         headerName: 'rules.grid.count',
         field: 'count'
       },
-      rulesColumnDefs.explore
+      {
+        ...rulesColumnDefs.explore,
+        cellRendererFramework: props => <LinkRenderer {...props} to={`/maintenance/rule/${props.value}`} onLinkClick={this.logExploreClick} />
```

Comment:
Why not add this in the xxConfig.js file? Don't you want to log whenever this link is clicked?

Reply:
discussed offline